src/main/java/edu/cornell/kfs/sys/CUKFSConstants.java

```diff
@@ -22,6 +22,15 @@
     public static final String DATE_FORMAT_mm_dd_yyyy_hh_mm_ss_am = "MM/dd/yyyy hh:mm:ss a";
     public static final String DATE_FORMAT_yyyy_MM_dd_T_HH_mm_ss_SSS = "yyyy-MM-dd'T'HH:mm:ss.SSS";
     public static final String DATE_FORMAT_dd_MMM_yy = "dd-MMM-yy";
+    public static final String DATE_FORMAT_yyyyMMddHHmmss = "yyyyMMddHHmmss";
+    public static final String DATE_FORMAT_yyyy_MM = "yyyy/MM";
+    public static final String DATE_FORMAT_yyyyMMdd_HHmmss = "yyyyMMdd_HHmmss";
+    public static final String DATE_FORMAT_yyyyMMdd_HHmmssSSS = "yyyyMMdd_HHmmssSSS";
+    public static final String DATE_FORMAT_MMddyyyy_hhmmss = "MMddyyyy_hhmmss";
+    public static final String DATE_FORMAT_MM_dd_yyyy = "MM/dd/yyyy";
```

Comment:
I think this is the same as KFSConstants.MONTH_DAY_YEAR_DATE_FORMAT

Reply:
Yes it is. Adjusting for that change.